ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 3.1.25 (in development)
 -----------------------
+- In non-optimizing builds we emscripten will now place the stack first in
+  memory, before global data.  This is to get more accurate stack overflow
+  errors (since overflow will trap rather currpting global data first).  Most
+  programs should not be effected by this change. (#18154)
```

Comment:
Why "most"? Shouldn't this be NFC?

Reply:
If your programs does weird stuff such as comparing stack and data pointers.. its conceivable your could notice.   I couldn't come up with a better way to phrase it without sounding scary.    Maybe I should just be very explicit.